README.md

```diff
@@ -1 +1,109 @@
-# domain-primitives-java
\ No newline at end of file
+# Domain Primitives Java
+
+![License](https://img.shields.io/hexpm/l/plug)
+
+Primitive, self-validating immutable object types.
+
+*Invented by [Stefan Ludwig](https://github.com/slu-it)*
```

Comment:
"Invented" is a little bit too strong of a word for me.

How about:
`As proposed by [Stefan Ludwig](https://github.com/slu-it) in his [strongly-typed-vs-javax-validation](https://github.com/slu-it/technology-examples/tree/master/strongly-typed-vs-javax-validation) prototype.`